src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    private static final int DEFAULT_GROWTH_FACTOR = 2;
+    private static final int DEFAULT_INITIAL_SIZE = 0;
+    private int size;
+    private Node<K, V>[] table;
```

Comment:
somebody was told me, that we needed to make space after our constant and not constant variable declarations.

Reply:
WDYM?